app-tasks/rf/src/rf/uploads/geotiff/create_thumbnails.py

```diff
@@ -0,0 +1,142 @@
+"""Creates thumbnails for geotiff imagery
+
+Depends on having geotiff file already downloaded
+"""
+import os
+import subprocess
+
+import boto3
+
+from rf.models import Thumbnail
+
+from .io import get_geotiff_dimensions
+
+def create_thumbnails(tif_path, scene_id, organization_id):
+    """Creates thumbnails based on a geotiff path and scene
+
+    Args:
+        tif_path (str): path of local tiff file
+        scene_id (str): uuid of the scene
+
+    Returns:
+        List[Thumbnail]
+    """
+
+    r_tif_path = os.path.join(
+        os.path.dirname(tif_path),
+        'resampled_{}'.format(os.path.basename(tif_path))
+    )
+    rp_tif_path = os.path.join(
+        os.path.dirname(tif_path),
+        'reprojected_{}'.format(os.path.basename(tif_path))
+    )
+
+    dim = get_geotiff_dimensions(tif_path)
+    max_dim = float(max(get_geotiff_dimensions(tif_path)))
```

Comment:
Is this a change? Good call if so, it was on my initial list of things to check but I think I forgot.